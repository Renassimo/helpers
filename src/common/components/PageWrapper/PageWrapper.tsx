import styled from '@emotion/styled';
import { css } from '@emotion/react';

const PageWrapper = styled.main(
  ({ theme }) => css`
    min-height: 100vh;
    background: linear-gradient(
      0deg,
      rgba(255, 254, 252, 1) 0%,
      rgba(223, 232, 255, 1) 100%
    );
    display: grid;
    grid-template-rows: ${theme.mixins.toolbar.minHeight}px 1fr;

    ${theme.breakpoints.down('md')} {
      grid-template-rows: calc(100vh - ${theme.mixins.toolbar.minHeight}px) ${theme
          .mixins.toolbar.minHeight}px;
    }
  `
);

export default PageWrapper;
