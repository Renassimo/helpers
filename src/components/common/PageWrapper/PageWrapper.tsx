import styled from "@emotion/styled";

const PageWrapper = styled.main`
  min-height: 100vh;
  background: linear-gradient(
    0deg,
    rgba(255, 254, 252, 1) 0%,
    rgba(223, 232, 255, 1) 100%
  );
`;

export default PageWrapper;