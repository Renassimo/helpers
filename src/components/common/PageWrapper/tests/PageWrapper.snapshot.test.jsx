import { render } from '@testing-library/react';
import PageWrapper from '../PageWrapper';

describe('PageWrapper', () => {
  test('renders successfully', () => {
    // Arrange
    // Act
    const { container } = render(<PageWrapper>Children</PageWrapper>);
    // Assert
    expect(container).toMatchSnapshot();
  });
});
