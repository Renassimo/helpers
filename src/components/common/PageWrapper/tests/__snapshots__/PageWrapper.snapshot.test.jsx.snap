// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageWrapper renders successfully 1`] = `
<div>
  <main
    class="css-hfxpxn"
  >
    Children
  </main>
</div>
`;
